internal/usecase/interfaces/tag.go

```diff
@@ -25,4 +25,6 @@ type CreateTagGroupParam struct {
 type Tag interface {
 	CreateItem(context.Context, CreateTagItemParam, *usecase.Operator) (*tag.Item, error)
 	CreateGroup(context.Context, CreateTagGroupParam, *usecase.Operator) (*tag.Group, error)
+	AttachItemToGroup(context.Context, id.TagID, id.TagID, *usecase.Operator) (*tag.Group, error)
+	DetachItemFromGroup(context.Context, id.TagID, id.TagID, *usecase.Operator) (*tag.Group, error)
```

Comment:
I think it's hard to understand which is group arg and which is item arg from signature. I recommend creating Attach/DetachItemToGroupParam struct.

```suggestion
	AttachItemToGroup(context.Context, AttachItemToGroupParam, *usecase.Operator) (*tag.Group, error)
	DetachItemFromGroup(context.Context, DetachItemToGroupParam, *usecase.Operator) (*tag.Group, error)
```